.github/workflows/buildAndTest.yml

```diff
@@ -66,15 +66,17 @@ jobs:
           -DTORCH_MLIR_USE_INSTALLED_PYTORCH=${{ matrix.torch-binary }} \
           -DCMAKE_OSX_ARCHITECTURES=${{ matrix.taregetarch }} \
           -DMACOSX_DEPLOYMENT_TARGET=10.15 \
-          -DLLVM_TARGETS_TO_BUILD=host
+          -DLLVM_TARGETS_TO_BUILD=host \
+          $GITHUB_WORKSPACE/externals/llvm-project/llvm
+        cmake --build build --target tools/torch-mlir/all
```

Comment:
This is the in-tree build step that is now co-located with the cmake configuration for in-tree.